src/main/java/com/nhantran/models/User.java

```diff
@@ -57,48 +57,36 @@ public void setPid(String pid) {
     }
 
     private static User getLoginAccountFromJsonFile(String accountName) {
-        JSONObject jsonObject = JsonFileReader.readJsonFromFile(Constants.ACCOUNT_JSON_FILE_PATH);
-        if (jsonObject != null) {
-            JSONObject accounts = (JSONObject) jsonObject.get("loginAccounts");
-            JSONObject userObj = (JSONObject) ((JSONArray) accounts.get(accountName)).get(0);
-            return new User((String) userObj.get("email"), (String) userObj.get("password"));
-        } else {
-            return null;
-        }
+        JSONObject userObj = JsonFileReader.readUserAccountFromFile("loginAccounts", accountName);
+        return new User((String) userObj.get("email"), (String) userObj.get("password"));
     }
 
     private static User getRegisterAccountFromJsonFile(String accountName) {
-        JSONObject jsonObject = JsonFileReader.readJsonFromFile(Constants.ACCOUNT_JSON_FILE_PATH);
-        if (jsonObject != null) {
-            JSONObject accounts = (JSONObject) jsonObject.get("registerAccounts");
-            JSONObject userObj = (JSONObject) ((JSONArray) accounts.get(accountName)).get(0);
-            return new User((String) userObj.get("email"), (String) userObj.get("password"), (String) userObj.get("confirmPassword"), (String) userObj.get("pid"));
-        } else {
-            return null;
-        }
+        JSONObject userObj = JsonFileReader.readUserAccountFromFile("registerAccounts", accountName);
+        return new User((String) userObj.get("email"), (String) userObj.get("password"), (String) userObj.get("confirmPassword"), (String) userObj.get("pid"));
     }
 
-    public static User getValidUser(){
+    public static User getValidUser() {
         return getLoginAccountFromJsonFile("validAccount");
     }
 
-    public static User getInvalidUser(){
+    public static User getInvalidUser() {
         return getLoginAccountFromJsonFile("invalidAccount");
     }
 
-    public static User getBlankEmailUser(){
-        return getLoginAccountFromJsonFile("blankEmailAccount");
+    public static User getBlankEmailUser() {
+        return getLoginAccountFromJsonFile("accountWithBlankEmail");
     }
 
-    public static User getInactivatedUser(){
+    public static User getInactivatedUser() {
```

Comment:
i think you call directly to get login account instead of creating each function for each case